Makefile

```diff
@@ -1,7 +1,35 @@
 GO_REQUIRED_MIN_VERSION = 1.16
 
+OPERATOR_VERSION := 1.0.0
+
 MANIFEST_SOURCE := https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.yaml
 
+APP_NAME=cert-manager-operator
+
+IMAGE_REGISTRY :=registry.svc.ci.openshift.org
+
+BUNDLE_IMAGE_NAME=cert-manager-operator-bundle
+BUNDLE_IMAGE_PATH=$(IMAGE_REGISTRY)/$(BUNDLE_IMAGE_NAME)
+BUNDLE_IMAGE_TAG?=latest
+
+# Detect the OS to set per-OS defaults
+OS_NAME=$(shell uname -s)
+# Container runtime
+ifeq ($(OS_NAME), Linux)
```

Comment:
This part is copy--pasted from the Compliance Operator. Is there any better way to do it?